examples/imu/calibrate.cc

```diff
@@ -0,0 +1,70 @@
+/****************************************************************************
+ *                                                                          *
+ *  Copyright (C) 2020 RoboMaster.                                          *
+ *  Illini RoboMaster @ University of Illinois at Urbana-Champaign          *
+ *                                                                          *
+ *  This program is free software: you can redistribute it and/or modify    *
+ *  it under the terms of the GNU General Public License as published by    *
+ *  the Free Software Foundation, either version 3 of the License, or       *
+ *  (at your option) any later version.                                     *
+ *                                                                          *
+ *  This program is distributed in the hope that it will be useful,         *
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of          *
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the           *
+ *  GNU General Public License for more details.                            *
+ *                                                                          *
+ *  You should have received a copy of the GNU General Public License       *
+ *  along with this program. If not, see <http://www.gnu.org/licenses/>.    *
+ *                                                                          *
+ ****************************************************************************/
+
+#include <cstring>
+
+#include "bsp_gpio.h"
+#include "bsp_imu.h"
+#include "bsp_os.h"
+#include "bsp_uart.h"
+#include "cmsis_os.h"
+#include "main.h"
+
+#define ONBOARD_IMU_SPI hspi5
+#define ONBOARD_IMU_CS_GROUP GPIOF
+#define ONBOARD_IMU_CS_PIN GPIO_PIN_6
+#define PRING_UART huart8
+
+typedef struct {
+  char header;
+  bsp::vec3f_t acce;
+  bsp::vec3f_t gyro;
+  bsp::vec3f_t mag;
+  char terminator;
+} __attribute__((packed)) imu_data_t;
+
+static bsp::MPU6500* imu;
+static bsp::UART* uart;
```

Comment:
Possible output via USB

Reply:
Changed UART to USB. I think print should be reserved for other things if we try to debug these examples.